signers/src/eth_signer/json_rpc_signer.rs

```diff
@@ -71,9 +74,12 @@ impl JsonRpcSigner {
         let params = serde_wasm_bindgen::to_value(&req_params)
             .map_err(|e| EthSignerError::CustomError(e.to_string()))?;
         let signature = self.provider.request(params).await.map_err(|e| {
-            EthSignerError::SigningFailed(serde_wasm_bindgen::from_value::<String>(e).unwrap())
+            EthSignerError::SigningFailed(
+                serde_wasm_bindgen::from_value::<String>(e).unwrap_or_default(),
+            )
```

Comment:
`EthSignerError::SigningFailed(e.to_string())`

Reply:
the error here is a JsValue type,can't call to_string() directly